Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN python -c "import duckdb; duckdb.connect().sql(\"INSTALL httpfs\");"
 
 FROM ubuntu:22.04 AS runner-image
 
-RUN apt-get update && apt-get install --no-install-recommends -y python3.11 python3.11-venv msodbcsql18 && \
+RUN apt-get update && apt-get install --no-install-recommends -y python3.11 python3.11-venv && \
```

Comment:
I'm confused. Why did you remove this here?

Reply:
I guess I misunderstood you; I thought we didn't want it included in the dockerfile but only in the git workflow?

Reply:
I'm fine with it being in the Dockerfile as we'll ship the CLI as a docker container, and it should come prepacked with necessary software packages that it works out of the box.